src/main/scala/common/Syntax.scala

```diff
@@ -224,6 +225,7 @@ object Syntax {
   case class Prog(
     includes: List[Include],
     defs: List[Definition],
+    decors: List[String],
```

Comment:
Can this be `List[CDecorate]` to be consistent?

Reply:
You're right; that's definitely better.